users/user-manager.js

```diff
@@ -68,8 +68,7 @@ var getUser = (id) => {
   return new response.ResponseObject(200, User);
```

Comment:
Put a TODO here to indicate that this needs to be hooked up to the `user-manager`